.gitignore

```diff
@@ -1,60 +1,120 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
 *.py[cod]
+*$py.class
 
 # C extensions
 *.so
 
 # Distribution / packaging
 .Python
-/env/
-/bin/
-/build/
-/develop-eggs/
-/dist/
-/eggs/
-/lib/
-/lib64/
-/parts/
-/sdist/
-/var/
-/*.egg-info/
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
```

Comment:
I would like to vote for not doing unrelated changes to `.gitignore` in here but in a dedicated pull request. Also, the leading slashes should probably be kept since the have meaning and make things go faster (because they apply to fewer places). I should add, that personally I try to keep .gitignore files minimal, i.e. I would not add entries before someone actually hits them. What I see below, seems to fall into that bucket.